src/index.css

```diff
@@ -33,23 +41,46 @@ nav a {
   color: #2d99f1;
 }
 
+.book-column {
+  padding: 0 2.5em;
+}
+
+.container {
+  margin-top: 3em;
+}
+
+.item-container {
+  display: flex;
+  align-items: center;
+  background-color: white;
+  padding: 0.8em;
+  margin-bottom: 1.3em;
+  border: 1px solid rgb(220, 220, 220, 0.7);
+  border-radius: 0.3em;
+}
```

Comment:
- [ ] I will kindly suggest you work on the alignment of the books and the form as seen in the original design.
- [ ] Also in the original design there is a profile icon at the top right hand side of the page, currently the profile icon is missing in your design. Kindly check out where i have highlighted in the image below to make correction.


| Your Design 👇 | Zeplin Design 👇 |
| ------------- | ------------- |
| ![book](https://user-images.githubusercontent.com/78634841/231898544-1696b4a9-d516-40eb-af4f-c824713c8ae4.png)| ![Bookstore CMS](https://user-images.githubusercontent.com/78634841/231898569-f0019659-c733-4ae0-942b-1f9e0c1bc6b7.png)|

Reply:
Hi  @brainconnect93 
Sweet in the middle😅
Duly noted👍